addons/stock/tests/test_proc_rule.py

```diff
@@ -21,6 +21,18 @@ def setUpClass(cls):
         })
         cls.partner = cls.env['res.partner'].create({'name': 'Partner'})
 
+    def test_qty_to_order_remainder_decimal(self):
+        """Test case for when remainder is decimal"""
+        self.env.user.groups_id += self.env.ref('stock.group_stock_multi_locations')
+        orderpoint_form = Form(self.env['stock.warehouse.orderpoint'])
+        orderpoint_form.product_id = self.product
+        orderpoint_form.location_id = self.env.ref('stock.stock_location_stock')
+        orderpoint_form.product_min_qty = 4.0
+        orderpoint_form.product_max_qty = 5.1
+        orderpoint_form.qty_multiple = 0.1
+        orderpoint = orderpoint_form.save()
+        self.assertEqual(orderpoint.qty_to_order, orderpoint.product_max_qty)
```

Comment:
Failing test was due to `orderpoint_form.product_max_qty` being checked instead of `orderpoint.product_max_qty`. Don't know why :man_shrugging:, but hopefully it won't be necessary to change in the previous stable versions since it passed in those/they're already merged